File: Assets/Scripts/CSharp/PopUpIcons.cs
using UnityEngine;
using System.Collections;

public class PopUpIcons : MonoBehaviour 
{	
	public float i;
	public float timeDif;
	public bool pop;
	public Vector3 old;
	public Vector3 newScale;
	
	IEnumerator Start () 
	{
		old = transform.localScale;
		yield return new WaitForSeconds(timeDif*i);
		pop = true;
	}
	
	void Update () 
	{
		if( pop )
		{
			transform.localScale = Vector3.Slerp(transform.localScale, newScale, Time.deltaTime * 6);
		}	
	}
}

File: Assets/Scripts/PopUpIcons.js
#pragma strict
var i:float;
var timeDif:float;
var pop:boolean;
var old:Vector3;
var newScale:Vector3;
function Start () {
	old = transform.localScale;
	yield WaitForSeconds(timeDif*i);
	pop = true;
}

function Update () {
	if(pop)
	{
		transform.localScale = Vector3.Slerp(transform.localScale, newScale, Time.deltaTime * 6);
	}	
}